doc/user/content/quickstarts/abandoned-cart.md

```diff
@@ -0,0 +1,332 @@
+---
+title: "Abandoned Cart"
+description: "Abandoned Cart Demo"
+menu:
+    main:
+        parent: quickstarts
+weight: 20
```

Comment:
```suggestion
weight: 20
draft: true
```

Reply:
Let's mark these as drafts so we can merge them ahead of the demo infra being up and running. CC @dehume @joacoc